Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3594,6 +3602,15 @@ def gen2021HiMix(fragment,howMuch):
                                       '--eventcontent':'FEVTDEBUGHLT',
                                       '--geometry' : geom
                                       }
+                                      
+    upgradeStepDict['HLT'][k] = {'-s':'HLT:%s'%(hltversion),
+                                 '--conditions':gt,
+                                 '--datatier':'GEN-SIM-DIGI-RAW',
+                                 '-n':'10',
+                                 '--eventcontent':'FEVTDEBUGHLT',
+                                 '--geometry' : geom,
+                                 '--outputCommands' : '"drop *_*_*_GEN,drop *_*_*_DIGI2RAW"'
```

Comment:
Thanks @jfernan2 ! You can reproduce the DQM crash by replacing the drop statements here with:
`"drop *_*_*_GEN,drop *_*_*_SIM,drop *_*_*_DIGI2RAW"` and running 12424.0. The error message at the last step is:

A fatal system signal has occurred: segmentation violation
The following is the call stack containing the origin of the signal.

Sun Apr 17 20:58:01 CEST 2022
Thread 2 (Thread 0x7f071393e700 (LWP 1782) "cmsRun"):
#0  0x00007f073c5e41d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00007f07362665e7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginFWCoreServicesPlugins.so
#2  0x00007f073626712a in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginFWCoreServicesPlugins.so
#3  0x00007f073cbe2bf4 in std::execute_native_thread_routine (__p=0x7f07323e4600) at ../../../../../libstdc++-v3/src/c++11/thread.cc:82
#4  0x00007f073c5dcea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00007f073c305b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x7f073a458540 (LWP 1577) "cmsRun"):
#0  0x00007f073c2faddd in poll () from /lib64/libc.so.6
#1  0x00007f073626689f in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginFWCoreServicesPlugins.so
#2  0x00007f07362671fc in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginFWCoreServicesPlugins.so
#3  0x00007f0736269a3b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00007f06ee55dcbc in EcalCondObjectContainer<EcalChannelStatusCode>::find(unsigned int) const () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libDQMEcalMonitorClient.so
#6  0x00007f06ee55ba65 in ecaldqm::IntegrityClient::producePlots(ecaldqm::DQWorkerClient::ProcessType) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libDQMEcalMonitorClient.so
#7  0x00007f06ee5bb5b4 in EcalDQMonitorClient::runWorkers(dqm::implementation::IGetter&, ecaldqm::DQWorkerClient::ProcessType) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginDQMEcalMonitorClientPlugins.so
#8  0x00007f06ee5bbf8d in EcalDQMonitorClient::dqmEndJob(dqm::implementation::IBooker&, dqm::implementation::IGetter&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginDQMEcalMonitorClientPlugins.so
#9  0x00007f06ee5bea34 in non-virtual thunk to DQMEDHarvester::endProcessBlockProduce(edm::ProcessBlock&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/pluginDQMEcalMonitorClientPlugins.so
#10 0x00007f073ed9f1e0 in edm::one::EDProducerBase::doEndProcessBlock(edm::ProcessBlockPrincipal const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#11 0x00007f073ed88a80 in edm::WorkerT<edm::one::EDProducerBase>::implDoEndProcessBlock(edm::ProcessBlockPrincipal const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#12 0x00007f073ec95567 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3> >(edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#13 0x00007f073ec95960 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3> >(std::__exception_ptr::exception_ptr const*, edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#14 0x00007f073ec95f0a in void edm::SerialTaskQueueChain::actionToRun<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#15 0x00007f073ec95fe1 in edm::SerialTaskQueue::QueuedTask<edm::SerialTaskQueueChain::push<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&>(tbb::detail::d1::task_group&, edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::ProcessBlockPrincipal, (edm::BranchActionType)3> >::execute()::{lambda()#1}&)::{lambda()#1}>::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#16 0x00007f073eee6055 in tbb::detail::d1::function_task<edm::SerialTaskQueue::spawn(edm::SerialTaskQueue::TaskBase&)::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreConcurrency.so
#17 0x00007f073d444a59 in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x7f06b7054c00, this=0x7f0738eafe00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_3_0-slc7_amd64_gcc11/build/CMSSW_12_3_0-build/BUILD/slc7_amd64_gcc11/external/tbb/v2021.4.0-0929d4245541a9360696e439234c1bfc/tbb-v2021.4.0/src/tbb/task_dispatcher.h:322
#18 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x7f0738eafe00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_3_0-slc7_amd64_gcc11/build/CMSSW_12_3_0-build/BUILD/slc7_amd64_gcc11/external/tbb/v2021.4.0-0929d4245541a9360696e439234c1bfc/tbb-v2021.4.0/src/tbb/task_dispatcher.h:463
#19 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_3_0-slc7_amd64_gcc11/build/CMSSW_12_3_0-build/BUILD/slc7_amd64_gcc11/external/tbb/v2021.4.0-0929d4245541a9360696e439234c1bfc/tbb-v2021.4.0/src/tbb/task_dispatcher.cpp:168
#20 0x00007f073ec624c3 in edm::EventProcessor::endProcessBlock(bool, bool) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#21 0x00007f073ec667f9 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc11/cms/cmssw/CMSSW_12_3_0/lib/slc7_amd64_gcc11/libFWCoreFramework.so
#22 0x000000000040a18d in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#23 0x00007f073d432898 in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_3_0-slc7_amd64_gcc11/build/CMSSW_12_3_0-build/BUILD/slc7_amd64_gcc11/external/tbb/v2021.4.0-0929d4245541a9360696e439234c1bfc/tbb-v2021.4.0/src/tbb/arena.cpp:698
#24 0x000000000040afd9 in main::{lambda()#1}::operator()() const ()
#25 0x00000000004096fc in main ()

Current Modules:

Module: EcalDQMonitorClient:ecalMonitorClient (crashed)